executor.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/indiependente/autoEqMac/server"
+)
+
+// NewExecutor returns a new prompt.Executor.
+func NewExecutor(srv *server.HTTPServer) func(string) {
+	return func(s string) {
+		s = strings.TrimSpace(s)
+		if s == "bye" || s == "quit" {
```

Comment:
empty string?

Reply:
better to just do nothing, could be a mistake